client/src/App.js

```diff
@@ -17,25 +20,33 @@ const AppWrapper = styled.div`
 `;
 
 function App() {
+  const [theme, setTheme] = useState(light);
+
+  const handleChangeTheme = () => {
+    setTheme(theme === dark ? light : dark);
+  };
+
```

Comment:
i saw that in an article, mabye that helps you ...

https://jacobdcastro.com/tutorials/how-to-use-themes-with-styled-components

```suggestion
const [isDarkTheme, toggleTheme] = useState(true);

  const action = () => {
    toggleTheme((isDarkTheme = !isDarkTheme));
  };
```